zul/src/archive/web/js/zul/inp/ComboWidget.js

```diff
@@ -595,8 +595,8 @@ zul.inp.ComboWidget = zk.$extends(zul.inp.InputWidget, {
 			return;
 		}
 
-		if (keyCode == 18 || keyCode == 27 || keyCode == 13
-		|| (keyCode >= 112 && keyCode <= 123)) //ALT, ESC, Enter, Fn
+		if (keyCode == 18 || keyCode == 27 || keyCode == 13 || keyCode == 93 || keyCode == 17
```

Comment:
> Other non content relevant keypresses also trigger autodrop (alt, numlock, context menu key, volume keys if any, insert key, pause key, etc.)

So it is not a good idea just add blacklist (try `print screen`)

> Possible improvement could be to use an onInput or onChange

use these events to open might be better

Reply:
ok